nexus/types/src/external_api/views.rs

```diff
@@ -1490,6 +1491,18 @@ pub struct TargetRelease {
     pub release_source: TargetReleaseSource,
 }
 
+/// Trusted root role used by the update system to verify update repositories.
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize, JsonSchema)]
+pub struct UpdatesTrustRoot {
+    /// The UUID of this trusted root role.
+    pub id: TufTrustRootUuid,
```

Comment:
So my impression was that we don't expose omicron UUID kinds in the external API -- though I could be wrong here.

Reply:
Most UUIDs don't seem to be exposed, but we haven't been consistent about that.

https://github.com/oxidecomputer/omicron/blob/17e4c0e5b0b697c63bd05d267c0fdaca2b2e79d7/openapi/nexus.json#L25291-L25306